notebooks/EDA.ipynb

```diff
@@ -0,0 +1,4712 @@
+{
```

Comment:
<p>Korzystamy gdzieś dalej z tej zamiany na <code>category</code>?  Z czego wynika, że akurat do 53 unikalnych zamieniamy typ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataWorkshop-Foundation/way-to-school/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
zmieniłam bo mniej waży dataset :)